app/html/index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Safe Browser Extension</title>
   </head>
-  <body style="width:400px; height:500px;">
+  <body style="width:370px; height:550px;">
```

Comment:
Does it work? I thought the style prop should be an object, of that is outside the React tree?